docs/content/latest/explore/security/security.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Security
+headerTitle: Security
+linkTitle: Security
+description: Overview of security in YugabyteDB.
+headcontent: Security in YugabyteDB.
+image: /images/section_icons/index/secure.png
+aliases:
+  - /latest/explore/security/
+menu:
+  latest:
+    identifier: explore-security
+    parent: explore
+    weight: 930
+isTocNested: true
+showAsideToc: true
+---
+
+Like PostgreSQL, YugabyteDB provides security in multiple ways:
+
+* **Authentication** - limit access to the database to clients with proper credentials
+
+* **Authorization** - create users and roles, and grant privileges to restrict activities that the users and roles can perform
+
+* **Encryption** - encrypt the database, as well as all network communication between servers
+
+* **Auditing** - conduct session- and object-level auditing
+
+* **Restricting network access** - limit connections to the database using RPC binding
+
+## Authentication
+ 
+Using client authentication, you can define how the database server establishes the identity of the client, and whether the client application (or the user who runs the client application) is allowed to connect with the database user name that was requested. YugabyteDB offers a number of different client authentication methods, including the following:
+
+* Password - authenticate using MD5 or SCRAM-SHA-256
+* LDAP - use external LDAP services to perform client authentication
+* Host-based - authenticate local and remote clients based on IP address and using TLS certificates
+* Trust - authorize specific local connections
+
+The method used to authenticate a particular client connection can be selected on the basis of (client) host address, database, and user.
+
+The authentication credentials in YugabyteDB are stored internally in the YB-Master system tables. The authentication mechanisms available to clients depends on what is supported and exposed by the YSQL, YCQL, and YEDIS APIs.
+
+Read more about [how to enable authentication in YugabyteDB](../../secure/authentication).
+
+## Authorization
+
+YugabyteDB provides role-based access control, consisting of a collection of privileges on resources given to roles. 
+
+Read more about [authorization in YugabyteDB](../../secure/authorization).
+
+### Roles
+
+Roles are essential to implementing and administering access control on a YugabyteDB cluster. Roles can represent individual users or a group of users, and encapsulate a set of privileges that can be assigned to other roles (or users). Roles can be modified to grant users or applications only the essential privileges based on the operations they need to perform against the database. Typically, an administrator role is created first. The administrator then creates additional roles for users.  
```

Comment:
You're going to get tired of me editing your stuff to remove passive and put it in second person. 🤓

```suggestion
Roles are essential to implementing and administering access control on a YugabyteDB cluster. Roles can represent individual users or a group of users, and encapsulate a set of privileges that can be assigned to other roles (or users). You can modify roles to grant users or applications the minimum required privileges based on the operations they need to perform against the database. Typically, you create an administrator role first, and the administrator then creates additional roles for users.  
```

Reply:
fixed